sklearn/metrics/_scorer.py

```diff
@@ -345,13 +352,16 @@ def get_scorer(scoring):
     """
     if isinstance(scoring, str):
         try:
-            scorer = SCORERS[scoring]
+            # make a deepcopy avoiding any change of the scorer object stored
+            # in SCORERS
+            scorer = deepcopy(SCORERS[scoring])
```

Comment:
I would argue that not deep copying `SCORERS[scoring]` was a bug. Since the objects in `SCORERS` are not heavy, I do not see any issues with using `deepcopy` here and call it a bug fix.